pages.pt-BR/windows/dir.md

```diff
@@ -0,0 +1,19 @@
+# dir
+
+> Lista os conteúdos de um diretório.
```

Comment:
Make sure to use the infinitive form in all verbs for all pages (here, for example: "Lista" --> "Listar", and below: "Mostra" --> "Mostrar").